src/main/java/appeng/spatial/CachedPlane.java

```diff
@@ -386,59 +390,64 @@ World getWorld()
 		return this.world;
 	}
 
-	private class Column
+	private static class BlockStorageData
 	{
+		public IBlockState state;
+		public int light;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make light a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)